src/main/java/at/hannibal2/skyhanni/utils/NEUVersionCheck.kt

```diff
@@ -35,21 +29,20 @@ object NEUVersionCheck {
             }
         } catch (_: Throwable) {
         }
-        neuWarning(
-            "NotEnoughUpdates is outdated!\n" +
-                "You currently need NEU version 2.2.0 or later.\n" +
-                "NEU 2.1 is NOT the latest version.\n" +
-                "Use these links to download the latest version:"
-        )
+        neuWarning("outdated")
     }
 
-    private fun neuWarning(text: String) {
+    private fun neuWarning(what: String) {
         openPopupWindow(
-            text,
+            "NotEnoughUpdates is $what!\n" +
+                "SkyHanni requires the latest version of NotEnoughUpdates to work.\n" +
+                "You currently need NEU version 2.2.0 or later.\n" +
```

Comment:
We might just not say any version here? So that we dont have to update this again and again.